gnomad_qc/v4/resources/annotations.py

```diff
@@ -398,3 +398,27 @@ def get_vrs(
             for version in VERSIONS
         },
     )
+
```

Comment:
I noticed that the intermediate subset files are being written to the 4day bucket because there is this `from gnomad_qc.v3.resources.basics import get_checkpoint_path` and the v3 checkpoint path goes to the 4day. 


I think we should first of all not import from that (sorry, I think it was me), and second change:

```
    if intermediate_subset:
        ht_name += f".{intermediate_subset}"
        if test:
            ht_name += ".test"
        ht_path = get_checkpoint_path(ht_name, version=CURRENT_VERSION)
```

to 
```
    if intermediate_subset:
        ht_name += f".{intermediate_subset}"
        if test:
            ht_name += ".test"
        ht_path = f"{_annotations_root(version, test)}/temp/{ht_name}.ht"
```

Reply:
Ah good catc, fixed!